tavla/app/(admin)/components/Preview/index.tsx

```diff
@@ -1,21 +1,112 @@
 'use client'
 
+import { LeftArrowIcon, RightArrowIcon } from '@entur/icons'
 import { Board } from 'Board/scenarios/Board'
-import { useEffect, useState } from 'react'
+import { usePostHog } from 'posthog-js/react'
+import { SetStateAction, useEffect, useState } from 'react'
 import { TBoard } from 'types/settings'
 
+const CarouselIndicators = ({
+    boards,
+    activeIndex,
+    onClick,
+}: {
+    boards: TBoard[]
+    activeIndex: number
+    onClick: (index: number) => void
+}) => {
+    return (
+        <div className="flex flex-row md:space-x-3 space-x-5 justify-center h-[10vh]">
+            {boards.map((_, index) => (
+                <button
+                    key={index}
+                    className={`md:w-5 md:h-5 w-6 h-6 rounded-full  bottom-5 mt-[2vh] left-1/2 ${
```

Comment:
vi kan heller sette` mt-6 `(eller en større/mindre verdi basert på hva spacingen egt skal være) i stedet for en arbitrary value som `2vh` på selve diven som wrapper disse knappene. vi trenger heller ikke `left-1/2` siden vi har satt `justify-center` på diven rundt